x/gov/keeper/deposit.go

```diff
@@ -92,3 +92,51 @@ func (keeper Keeper) GetDepositsIterator(ctx sdk.Context, proposalID uint64) sdk
 	store := ctx.KVStore(keeper.storeKey)
 	return sdk.KVStorePrefixIterator(store, types.DepositsKey(proposalID))
 }
+
+// GetAllDeposits returns all the deposits from the store
+func (keeper Keeper) GetAllDeposits(ctx sdk.Context) (deposits types.Deposits) {
+	keeper.IterateAllDeposits(ctx, func(deposit types.Deposit) bool {
+		deposits = append(deposits, deposit)
+		return false
+	})
+
+	return
+}
+
+// // RefundDeposits refunds and deletes all the deposits on a specific proposal
+// func (keeper Keeper) RefundDeposits(ctx sdk.Context, proposalID uint64) {
```

Comment:
any dependency for this?

Reply:
end_blocker
@murthyvitwit can you please explain about end blocker